assets/base/config.json

```diff
@@ -6,5 +6,16 @@
     "MinServerVersion": "4.0.0",
     "PlatformNoticeURL": "https://about.mattermost.com/platform-notice-txt/",
     "MobileNoticeURL": "https://about.mattermost.com/mobile-notice-txt/",
-    "SegmentApiKey": "3MT7rAoC0OP7yy3ThzqFSAtKzmzqtUPX"
+    "SegmentApiKey": "3MT7rAoC0OP7yy3ThzqFSAtKzmzqtUPX",
+
+    "SentryEnabled": false,
```

Comment:
should this be `true`?

Reply:
It's set to true and the other fields are filled in as part of the build process. I just left it false for development builds and anyone compiling it themselves

Reply:
got it, makes sense.